README.md

```diff
@@ -105,6 +105,10 @@ To include your `avatar`, You just have to replace `<yourusername>` with your **
 
 For example: <http://github.com/eddiejaoude.png>
 
+### A simple gif version of adding your profile
+
+<https://github.com/KarlGusta/LinkFree-docs/blob/main/Linkfree%20adding%20profile%20gif.gif>
```

Comment:
```suggestion
![Linkfree adding profile GIF](https://user-images.githubusercontent.com/51878265/165113896-35d34c1d-c95e-4bb4-b658-df1afb9ab658.gif)
```



__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
You can commit the suggestions